pkgs/applications/graphics/curv/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+, openexr
+, tbb
+, glm
+, libpng
+, eigen
+, libGL
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "curv";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "curv3d";
+    repo = "curv";
+    rev = "refs/tags/${version}";
+    hash = "sha256-m4p5uxRk6kEJUilmbQ1zJcQDRvRCV7pkxnqupZJxyjo=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    cmake
+    boost
+    openexr
+    tbb
+    glm
+    libpng
+    eigen
+    libGL
+    xorg.libX11
+    xorg.libXrandr
+    xorg.libXinerama
+    xorg.libXcursor
+    xorg.libXi
+    xorg.libXext
+  ];
+
+  # This test does not use the GPU
```

Comment:
Does this mean that upstream test cases uses GPU thus not run? Please make it clear here.